dotnet/samples/KernelSyntaxExamples/Example12_SequentialPlanner.cs

```diff
@@ -139,6 +142,51 @@ private static async Task EmailSamplesAsync()
             "They ruled the kingdom together, ruling with fairness and compassion, just as Arjun had done before. They lived " +
             "happily ever after, with the people of the kingdom remembering Mira as the brave young woman who saved them from the dragon.";
         await ExecutePlanAsync(kernel, plan, input, 5);
+
+        Console.WriteLine("======== Sequential Planner - Find and Execute Saved Plan ========");
+
+        // Save the plan for future use
+        var memoryStore = GetMemory();
```

Comment:
Creating a Kernel to get the `.Memory` property seems as an overkill.

Shouldn't that be just the `new VolatileMemoryStore()` instance directly?

```suggestion
        var memoryStore = new VolatileMemoryStore();
```

Reply:
It's not a memoryStore, so I should update the naming. It's a ISemanticTextMemory instance, which yeah, I could do with:

```
        var memoryStorage = new VolatileMemoryStore();
        var textEmbeddingGenerator = new AzureTextEmbeddingGeneration(
            modelId: azureOpenAIEmbeddingDeployment,
            endpoint: azureOpenAIEndpoint,
            apiKey: azureOpenAIKey,
            logger: logger);
        using var memory = new SemanticTextMemory(memoryStorage, textEmbeddingGenerator);
```